### PR TITLE
Scenarios: remove recipes from items list

### PR DESCRIPTION
**Pull request by Argn0**
no data for those

### CODE REVIEW

File: src/components/Scenarios/FormFieldData.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function getFormFieldData(metadata) {
     }))
       .sort((a, b) => a.text.localeCompare(b.text)),
 
-    itemList: Object.keys(items).map(k => [items[k], k]).filter(x => x[0].cost >= itemCost).map(x => ({
+    itemList: Object.keys(items).map(k => [items[k], k]).filter(x => x[0].cost >= itemCost && !x[1].includes('recipe_')).map(x => ({
```
Comment (howardchung) [inline src/components/Scenarios/FormFieldData.jsx]: indexOf !== 0 might be a little faster

Comment (howardchung) [inline src/components/Scenarios/FormFieldData.jsx]: maybe you could move the map after the filter to avoid the need to double map?

Comment (Argn0) [reply src/components/Scenarios/FormFieldData.jsx]: I'm not sure if that's possible. I need to filter by property name and value. The items are stored in an object and not an array.

Comment (Argn0) [reply src/components/Scenarios/FormFieldData.jsx]: I think startsWith() is even more appropriate to be used here?

Comment (howardchung) [reply src/components/Scenarios/FormFieldData.jsx]: you can still use items[k] to get the item object

Comment (Argn0) [reply src/components/Scenarios/FormFieldData.jsx]: whoops you're totally right. I had a total tunnel vision.

**Pull request merged**